packages/models-library/src/models_library/rabbitmq_messages.py

```diff
@@ -188,8 +188,11 @@ class RabbitResourceTrackingStartedMessage(RabbitResourceTrackingBaseMessage):
         default="tracking_started", const=True
     )
 
-    wallet_id: WalletID
-    wallet_name: str
+    wallet_id: WalletID | None
```

Comment:
why nullable?

Reply:
Yes, you are right we need to be careful with this.
1. Currently for simcore.io we will not have a concept of wallets, so it makes sense that we still collect usage information, but without wallets. (Alternatively, we can introduce some "general hidden wallet")
2. Just so we can merge it and start to collect data (I removed previously fixed numbers from the code such as -1 or 42 :) )